nautobot_chatops/dispatchers/slack.py

```diff
@@ -231,7 +231,8 @@ def send_snippet(self, text, title=None, ephemeral=None):
             if ephemeral:
                 message_list = self.split_message(text, SLACK_MAX_MESSAGE_LENGTH)
                 for msg in message_list:
-                    self.send_blocks(self.markdown_block(f"```\n{msg}\n```"), ephemeral=ephemeral)
+                    # Send the blocks as a list, this needs to be the case for Slack to send appropriately.
+                    self.send_blocks([self.markdown_block(msg)], ephemeral=ephemeral)
```

Comment:
Don't we still need to have the f-string to apply code fencing to `msg`?
```suggestion
                    self.send_blocks([self.markdown_block(f"```\n{msg}\n```")], ephemeral=ephemeral)
```